### ISSUE TITLE
[Potential] Host an Access Guide event

### ISSUE THREAD

**Issue by nickttng** [A11YChi/meetups-and-events#3]
**Why:** 

- As an attendee new to accessibility, I want help in getting started in accessibility. 
- As an attendee who is interested in facilitating accessibility workshop, I want to learn how I could use Access Guide as a tool

> As a tool, [Access Guide](https://github.com/thetuttingtutor/accessguide) is an introduction that takes the WCAG 2.1 success criteria and summarizes them into a collection of about 70 action items.

## ToDo

**30 days before the event**
- [x] Co-org to reach out to presenter
- [x] Have a kick-off conversation
  - [ ] Schedule a testing group before hosting the main event
- [ ] Coordinate with co-orgs for date availability for the main event
- [ ] Coordinate with the presenter for next steps
  - [ ] Get synopsis and bio
  - [ ] Set potential dates
  - [ ] Share 'Accessible Presentation' guideline 
- [ ] Secure a venue
- [ ] Secure ACS captioner (Note: We might need to adjust our approach)
- [ ] Create event page/URL

**15 days before the event**
- [ ] Announce through channels (MTLS)
  - [ ] Mailing list
  - [ ] Twitter
  - [ ] LinkedIn
  - [ ] Web-A11Y Slack
- [ ] Share event page/URL with venue and presenter
- [ ] If feasible, request (rough) slides from presenter for _initial_ accessibility check

**1-5 days before the event**
- [ ] Reminders through channels (MTLS)
  - [ ] Mailing list
  - [ ] Twitter
  - [ ] LinkedIn
  - [ ] Web-A11Y Slack
- [ ] Soft check with venue for anything needed
- [ ] Follow up with presenter for time of arrival (20-30 mins before door opens) and any needs
- [ ] Update housekeeping slides

**Day of event**
- [ ] Bring no-photo lanyards
- [ ] Bring pronoun buttons
- [ ] Bring A/TV equipment for streaming
- [ ] Arrive 30-45 minutes before door opens (Co-orgs meet the venue team)
- [ ] Coordinate with presenter
- [ ] Announcements and talk

**1-5 days after the event**
- [ ] Post-announcement, YouTube share, and thanks (MTLS)
  - [ ] Shoutout for presenter
  - [ ] Shoutout for venue
- [ ] Finalize transcrit
  - [ ] Get transcript from ACS
  - [ ] Edit transcript and push to GitHub transcript repo

**Post-event (based on team bandwidth)**
- [ ] Add transcript to YouTube video talk
